visuals/fill.py

```diff
@@ -0,0 +1,18 @@
+from .cube import *
+import random
+
+def fill():
+  c = Cube()
+  colour = Colour.cyan()
+  on = True
+  while True:
+    all_coords = [(x, y, z) for x in range(c.size) for y in range(c.size) for z in range(c.size)]
```

Comment:
nit: you could use Pos(x, y, z) here instead of below

Reply:
done!